core/src/types.ts

```diff
@@ -266,3 +266,53 @@ export declare interface DebtAuctionEnriched extends DebtAuctionActive {
 }
 
 export declare interface DebtAuctionTransaction extends DebtAuctionEnriched, CompensationAuctionTransactionFees {}
+
+type CollateralType = CollateralConfig['title'];
+export declare interface VaultCollateralType {
+    collateralType: CollateralType;
+}
+
+export declare interface LiquidationLimit {
+    maximumAmountAuctionedCollateral: BigNumber;
+    currentlyAuctionedCollateral: BigNumber;
+}
+
+export declare interface VaultCollateralParameters extends LiquidationLimit {
+    stabilityFeeRate: BigNumber;
+    maxDebtPerCollateralUnit: BigNumber;
```

Comment:
Is't `maxDebtPerCollateralUnit` also `*Rate` (not `*Unit`) or even `*Ratio` as in other keys below?

Reply:
No, i am pretty sure that it's the `perUnit` metric since the condition is `stabilityFeeRate * initialDebtDai > collateralAmount * maxDebtPerCollateralUnit(Rate)` https://github.com/makerdao/dss/blob/fa4f6630afb0624d04a003e920b0d71a00331d98/src/dog.sol#L181

and the documentation states / comment states that it's a "price" 
https://github.com/makerdao/dss/blob/fa4f6630afb0624d04a003e920b0d71a00331d98/src/vat.sol#L47

Reply:
So can `maxDebtPerCollateralUnit` be called `minUnitPrice`? because `debt`( `DAI`) per collateral (eg `ETH`) is basically unit price which is the minimum, lower which the vault will become `liquidatable`?